Tools/autotest/default_params/gazebo-iris.parm

```diff
@@ -0,0 +1,638 @@
+ATC_RAT_RLL_D    0.003600
+ATC_RAT_RLL_FILT 20.000000
+ATC_RAT_RLL_I    0.090000
+ATC_RAT_RLL_IMAX 0.444000
+ATC_RAT_RLL_P    0.135000
+ATC_RAT_YAW_D    0.000000
+ATC_RAT_YAW_FILT 5.000000
+ATC_RAT_YAW_I    0.018000
+ATC_RAT_YAW_IMAX 0.222000
+ATC_RAT_YAW_P    0.180000
+ATC_SLEW_YAW     6000.000000
+ATC_THR_MIX_MAX  0.500000
+ATC_THR_MIX_MIN  0.100000
+AUTOTUNE_AGGR    0.100000
+AUTOTUNE_AXES    7.000000
+AUTOTUNE_MIN_D   0.001000
+AVD_ENABLE       0.000000
+AVOID_ENABLE     1.000000
+BATT2_AMP_OFFSET 0.000000
+BATT2_AMP_PERVOL 17.000000
+BATT2_CAPACITY   3300.000000
+BATT2_CURR_PIN   12.000000
+BATT2_MONITOR    0.000000
+BATT2_VOLT_MULT  10.100000
+BATT2_VOLT_PIN   13.000000
+BATT_AMP_OFFSET  0.000000
+BATT_AMP_PERVOLT 17.000000
+BATT_CAPACITY    3300.000000
+BATT_CURR_PIN    12.000000
+BATT_MONITOR     4.000000
+BATT_VOLT_MULT   10.100000
+BATT_VOLT_PIN    13.000000
+BRD_SERIAL_NUM   0.000000
+BTN_ENABLE       0.000000
+CAM_DURATION     10.000000
+CAM_FEEDBACK_PIN -1.000000
+CAM_FEEDBACK_POL 1.000000
+CAM_MAX_ROLL     0.000000
+CAM_MIN_INTERVAL 0.000000
+CAM_RELAY_ON     1.000000
+CAM_SERVO_OFF    1100.000000
+CAM_SERVO_ON     1300.000000
+CAM_TRIGG_DIST   0.000000
+CAM_TRIGG_TYPE   0.000000
+CH10_OPT         0.000000
+CH11_OPT         0.000000
+CH12_OPT         0.000000
+CH7_OPT          7.000000
+CH8_OPT          0.000000
+CH9_OPT          0.000000
+CHUTE_ENABLED    0.000000
+CIRCLE_RADIUS    1000.000000
+CIRCLE_RATE      20.000000
+CLI_ENABLED      0.000000
+COMPASS_AUTODEC  1.000000
+COMPASS_CAL_FIT  8.000000
+COMPASS_DEC      0.000000
+COMPASS_DEV_ID   0.000000
+COMPASS_DEV_ID2  0.000000
+COMPASS_DEV_ID3  0.000000
+COMPASS_DIA2_X   1.000000
+COMPASS_DIA2_Y   1.000000
+COMPASS_DIA2_Z   1.000000
+COMPASS_DIA3_X   0.000000
+COMPASS_DIA3_Y   0.000000
+COMPASS_DIA3_Z   0.000000
+COMPASS_DIA_X    1.000000
+COMPASS_DIA_Y    1.000000
+COMPASS_DIA_Z    1.000000
+COMPASS_EXTERN2  0.000000
+COMPASS_EXTERN3  0.000000
+COMPASS_EXTERNAL 0.000000
+COMPASS_LEARN    0.000000
+COMPASS_MOT2_X   0.000000
+COMPASS_MOT2_Y   0.000000
+COMPASS_MOT2_Z   0.000000
+COMPASS_MOT3_X   0.000000
+COMPASS_MOT3_Y   0.000000
+COMPASS_MOT3_Z   0.000000
+COMPASS_MOTCT    0.000000
+COMPASS_MOT_X    0.000000
+COMPASS_MOT_Y    0.000000
+COMPASS_MOT_Z    0.000000
+COMPASS_ODI2_X   0.000000
+COMPASS_ODI2_Y   0.000000
+COMPASS_ODI2_Z   0.000000
+COMPASS_ODI3_X   0.000000
+COMPASS_ODI3_Y   0.000000
+COMPASS_ODI3_Z   0.000000
+COMPASS_ODI_X    0.000000
+COMPASS_ODI_Y    0.000000
+COMPASS_ODI_Z    0.000000
+COMPASS_OFS2_X   5.000000
+COMPASS_OFS2_Y   13.000000
+COMPASS_OFS2_Z   -18.000000
+COMPASS_OFS3_X   0.000000
+COMPASS_OFS3_Y   0.000000
+COMPASS_OFS3_Z   0.000000
+COMPASS_OFS_X    5.000000
+COMPASS_OFS_Y    13.000000
+COMPASS_OFS_Z    -18.000000
+COMPASS_ORIENT   0.000000
+COMPASS_ORIENT2  0.000000
+COMPASS_ORIENT3  0.000000
+COMPASS_PRIMARY  0.000000
+COMPASS_USE      1.000000
+COMPASS_USE2     1.000000
+COMPASS_USE3     1.000000
+DISARM_DELAY     10.000000
```

Comment:
these parameter files should only include non-default parameters as otherwise they bitrot very quickly as we change defaults or add/remove parameters